assets/Navbar/Ruchi/Navbar_code.txt

```diff
@@ -0,0 +1,87 @@
+<html>
```

Comment:
Could you please write the entire code in Reactjs Format? (Not in TXT)
if you don't know what is react , then Iet me know.

Reply:
Also no need to attach Navbar Image in the Assets folder.

Reply:
> Could you please write the entire code in Reactjs Format? (Not in TXT)
> if you don't know what is react , then Iet me know.

Apologize for the inconvenience caused by me. I didn't know about the tech stack for this issue, now after knowing  that I want to inform you that I don't know what is react.
Looking forward for your suggestion after this misconception....

Reply:
That's Fine. Do you want to work on it or I will assign some other participant.

Reply:
> That's Fine. Do you want to work on it or I will assign some other participant?
> If you want to learn to react then I will attach two link. You can start learning React From here.
> https://www.youtube.com/watch?v=Ke90Tje7VS0
> https://reactjs.org/

First of all thanks for your support. Please let me know that how much time you are expecting from me for required design. If you want this issue to be solved immediately then please assign this to other participant and if you can give me around two days then within that time I will re-design this using react with learning parallelly....
Thanks for your guidance....

Reply:
Ok as of now I am assigning this issue to some other participant.

Reply:
Ok, thanks for your precious time...